lib/lrama/grammar/parameterizing_rule_builder.rb

```diff
@@ -0,0 +1,47 @@
+module Lrama
+  class Grammar
+    class ParameterizingRuleBuilder
+      attr_reader :name, :args, :rhs, :term
+
+      def initialize(name, args, rhs)
+        @name = name
+        @args = args
+        @rhs = rhs
+        @term = nil
+        @required_args_count = args.count
+      end
+
+      def build_rules(token, build_token, rule_counter, lhs_tag, user_code, precedence_sym, line)
```

Comment:
`user_code` is defined by `%rule` and can not be overridden by caller. Then I think `user_code` argument is not needed.

Reply:
Indeed it is. Perhaps also `precedence_sym`. Removed in the following commit:
https://github.com/ruby/lrama/pull/285/commits/69b95e7cf6a75e4fbf80a73f3a3807cc841e605d